### PR TITLE
FEAT (#11) : update the /work-with-us page

### PR DESCRIPTION
**Pull request by activate-glacier-instinct**
Temporary PR for the /work-with-us work

### CODE REVIEW

**Comment by activate-glacier-instinct** [comment]
localhost_3000_work-with-us (2)

**Comment by activate-glacier-instinct** [comment]
Done:
- design tweaks on images, text, spacing
- all images uploaded
- all first links added

Todo:
- [x] Carousels are not working atm, cannot change slides to a new on eg. click on Academy the slide doesn't update
- [x] Update content for each service, once the carousel is working
- [x] Fix any spacing and padding to match designs

**Review by ken333135 — COMMENTED** [review]
Looks great to me!

**Comment by CompositeFellow** [comment]
I second this one

**Comment by activate-glacier-instinct** [comment]
Ready for a proper review now. 

Test URL: https://bankless-website-7jtdojlqm-activate-glacier-instinct.vercel.app/work-with-us

All above tasks complete:
- [x] carousels
- [x] content (as much as could scrounge)
- [x[ responsiveness
- [x] first pass image optimisations

Having trouble with all the content loading during Google Dev Tool screen capture - so images are missing. Best to checkout the deployed version above.

Screenshots:
desktop
tablet
mobile

**Comment by CompositeFellow** [comment]
minor change.  Can we update the bounty board link to go to https://www.bountyboard.xyz/ instead of https://bountyboard.bankless.community/ .  The community site will soon be obsolete

File: assets/content/pages/work-with-us/carouselContent.json [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+        "paras": [
+          "Bankless Consulting is the world's first web3-native consulting firm. Whether you're a DAO or a traditional enterprise, we'll help you capture the potential of the rapidly expanding decentralized economy."
+        ],
+        "cta": {
+          "text": "Bankless Consulting",
+          "link": "https://www.banklessconsulting.com/"
+        },
+        "image": {
+          "src": "/images/work-with-us/work-with-us-12.jpg",
+          "alt": "Photo by Jason Goodman on Unsplash"
```
Comment (ghost) [inline assets/content/pages/work-with-us/carouselContent.json:203]: credit to the photographer, well done!

**Comment by ghost** [comment]
Looks good to me

File: assets/content/pages/work-with-us/carouselContent.json (hunk 1 above)
Comment (activate-glacier-instinct) [reply assets/content/pages/work-with-us/carouselContent.json:203]: Not fully happy with this but yeah worth attributing where possible.

I'm thinking of changing it to a <figure> where the <figcaption> is the attribution

That way we can rewrite the alt as an actual alt :shrug:

**Pull request merged**